### ISSUE TITLE
W1b: `RandomPlasma*` do not replay bitwise from `params=` — the fractal is drawn in `apply_transform`

### ISSUE THREAD

**Issue by ducha-aiki** [kornia/kornia#4445]
`forward(x, params=aug._params)` replays a previous draw bitwise on every public augmentation class except the three `RandomPlasma*` classes. Their `_params` hold only the sampled `roughness` / `intensity` / `shade_*` factors; the plasma fractal itself is drawn inside `apply_transform` from torch's global generator on every call, so replaying with the stored parameters gives a different image unless the global seed is reset to the same value first. The class docstrings describe `params` the same way as every other class and do not say that the replay is partial.

## Repro

```python
import torch, kornia.augmentation as K

torch.manual_seed(0)
x = torch.rand(2, 3, 6, 8)
for cls in (K.RandomPlasmaBrightness, K.RandomPlasmaContrast, K.RandomPlasmaShadow, K.RandomBrightness):
    aug = cls(p=1.0)
    y1 = aug(x)
    y2 = aug(x, params=aug._params)
    torch.manual_seed(3); y3 = aug(x, params=aug._params)
    torch.manual_seed(3); y4 = aug(x, params=aug._params)
    print(f"{cls.__name__:24s} replay bitwise={torch.equal(y1, y2)}  same-global-seed replays equal={torch.equal(y3, y4)}  params={sorted(aug._params)}")
```

## Observed

```
RandomPlasmaBrightness   replay bitwise=False  same-global-seed replays equal=True  params=['batch_prob', 'forward_input_shape', 'intensity', 'roughness']
RandomPlasmaContrast     replay bitwise=False  same-global-seed replays equal=True  params=['batch_prob', 'forward_input_shape', 'roughness']
RandomPlasmaShadow       replay bitwise=False  same-global-seed replays equal=True  params=['batch_prob', 'forward_input_shape', 'roughness', 'shade_intensity', 'shade_quantity']
RandomBrightness         replay bitwise=True  same-global-seed replays equal=True  params=['batch_prob', 'brightness_factor', 'forward_input_shape']
```

## Expected

Either the fractal field is drawn in `generate_parameters` and stored in `_params` (so `params=` replays bitwise like every other class, at the cost of a larger parameter dict), or the three classes document that `params=` restores the factors only and that a bitwise replay additionally needs the global seed. The first outcome matches the contract the anchor documents for the rest of the package; the audit recommends it.

## Classification

**Semantic contract** — the `params=` replay contract holds on 66 of 69 concrete classes and silently does not on these three. Idiom: 5 (a documented precondition never validated: the docstring's `params` contract is not honoured).

## Disposition

`n/a — single symbol` (one family, three classes sharing one generator path). The batch documents the exception on the anchor and pins it as a wart; moving the draw into `generate_parameters` is a focused fix, welcome as a PR.

## Environment

kornia 0.9.0rc1 @ `d6a2c138` (main), torch 2.14.0, python 3.11, cpu. Found during the augmentation conventions audit (batch 6, tracking issue #4407); the batch documents the behavior above as it is and pins it, and does not change it.

Posted on behalf of @ducha-aiki by Claude (Fable 5.1).
